keyboards/annepro2/keymaps/thomazmoura/rules.mk

```diff
@@ -0,0 +1,47 @@
+# Anne Pro 2
```

Comment:
Is there a specific reason to include this? Having a lot of different make files could make any build change hard to maintain.